web-local/test/ui/utils/commonHelpers.ts

```diff
@@ -85,3 +85,11 @@ export async function waitForValidResource(
     }
   });
 }
+
+export async function waitForDashboard(page: Page) {
```

Comment:
`waitForAdbidsDashboard` to be more precise?

Reply:
this line actually fell out of this review when I updated to align with @AdityaHegde's PR. 

agree in principle with this renaming, we can adjust in subsequent PRs. (though i also thought that at the moment the adbids dashboard is the only dashboard we e2e test, to there is no ambiguity?)